9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
@hhorak how about instead of suggesting that users need to go build their own images on top of `centos/postgresql-95-centos7:latest` (more work for the use, can break when `:latest` image changes), we suggest to mount the alternative script in the appropriate path?

IOW, no need to write a custom Dockerfile, just run the image as:

```
docker run -v PATH/TO/MY/post-init:/usr/bin/hooks.d/post-init centos/postgresql-95-centos7
```

This approach is also supported in OpenShift.